go/carmen/carmen.go

```diff
@@ -277,6 +277,9 @@ type TransactionContext interface {
 	// that gets reset after each transaction.
 	SetTransientState(Address, Key, Value)
 
+	// HasEmptyStorage returns true if account has empty storage.
+	HasEmptyStorage(address Address) bool
```

Comment:
To reduce confusion between `State` and `Storage` we should rename this function to `HasEmptyState`.